demo/website/src/components/Project.js

```diff
@@ -1,25 +1,28 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
+import './Project.css';
+
 class Project extends Component {
   render() {
     return (
       <li className="ecl-list-item">
-        <a href="" className="ecl-link ecl-list-item__link">
+        <a
```

Comment:
I'm not sure the link to the `project_website` is always a true assumption. On the portal, they have inner pages.